src/screens/ProfileScreen/elements/BottomSheetBio.js

```diff
@@ -62,7 +67,7 @@ const styles = StyleSheet.create({
   description: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the description text has been changed from `COLORS.blackgrey` to `COLORS.gray400`. This change should improve readability and user experience by providing a more visually appealing contrast between the text and the background.
```diff
- color: COLORS.blackgrey,
+ color: COLORS.gray400,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->